insights/src/main/java/org/jboss/as/telemetry/extension/InsightsEnabledHandler.java

```diff
@@ -0,0 +1,45 @@
+package org.jboss.as.insights.extension;
```

Comment:
Please assume that comment is on every class file in the PR where it doesn't exist.